00-dev-environment/README.md

```diff
@@ -134,27 +134,27 @@ Which AWS environment variable cannot be set in order to run the
 `aws sts get-session-token` command?
 
 > It would seem the answer is `AWS_ACCESS_KEY_ID`, based on...
```

Comment:
It looks like in your logs, the get-session-token call succeeded when this was set.  There is a specific reason that we are going for here around the AWS_SESSION_TOKEN value, we can talk about this on the call this afternoon.